src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -550,6 +562,55 @@ private static bool ProcessAnyFileContent(IRequestInfo info, string localPath)
 			return true;
 		}
 
+		private bool ProcessCssFile(IRequestInfo info, string localPath)
+		{
+			// is this request the full path to a real file?
+			if (File.Exists(localPath) && Path.IsPathRooted(localPath))
```

Comment:
May be worth commenting here that this path will typically be taken for files in the book folder (since the browser supplies the preceding path). This is desirable for a couple of files. I thought though that there were files we copy to the book folder so it will be self-contained, but which we want to preferentially find a current version of in the files we ship?